packages/sprites/src/fs.ts

```diff
@@ -1,11 +1,11 @@
-import { readdir, readFile } from 'node:fs/promises';
+import { promises as fs } from 'node:fs';
```

Comment:
Just wondering what is the difference between `import { promises as fs } from 'node:fs';` and `import { promises as fs } from 'fs';` ?